comixed-importer/src/main/java/org/comixed/importer/ImportFileProcessor.java

```diff
@@ -74,7 +77,7 @@ public void setImportUser(String user) {
    *
    * @throws ProcessorException if a processing error occurs
    */
-  public void process(String source) throws ProcessorException {
+  public void process(String source) throws ProcessorException, ComicException {
```

Comment:
I don't think we should add the ComicException to the method signature. Instead, can we have the implementation catch the ComicException and rethrow it as a ProcessorException? That way we know each step of the way in the stacktrace where something bad happened.

Reply:
You're right. Changed it